ql/src/test/queries/clientpositive/analyze_npe.q

```diff
@@ -1,3 +1,4 @@
+--! qt:disabled:HIVE-24656
```

Comment:
I think this can be enabled now that HIVE-24656 has been committed?

Reply:
Yeap, I reenabled the test and rebased the PR.